atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableDefinition.java

```diff
@@ -343,7 +351,7 @@ public TableMetadata toTableMetadata() {
                 sweepStrategy,
                 expirationStrategy,
                 appendHeavyAndReadLight,
-                tableNameLoggable);
+                tableNameSafety.isSafeForLogging());
```

Comment:
Yep, initially went with the minimal change here.

That said I think it's fine to regenerate the relevant protobufs, no one should be using them yet (there's no way for people to set this yet, beyond putting specific byte[]s of metadata in any case!)